releasenotes/notes/use-docker-to-autodiscover-kubelet-host-ff2541815dabd0bc.yaml

```diff
@@ -0,0 +1,5 @@
+---
+enhancements:
+  - |
+    Resolve once the DNS name gave by docker and try the associated IP to reach the kubelet
```

Comment:
gave --> given. We could also make this more detailed, explaining briefly what the strategy is (https > http, IPs > hostnames), and that we're trying to resolve domain names to IPs to spare DNS servers across the cluster.